buku

```diff
@@ -4962,23 +4962,23 @@ def print_single_rec(row: BookmarkVar, idx: int=0, columns: int=0):  # NOQA
             return
 
         INDENT = 5
-        ln_num = 1
         fillwidth = columns - INDENT
+        desc_lines = [s for line in row.desc.splitlines()
+                        for s in textwrap.wrap(line, width=fillwidth) or ['']]
```

Comment:
Splitting into lines then wrapping each separately (the fallback value is for empty lines since `wrap()` swallows them)